src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -889,6 +889,7 @@ class XrdAccSciTokens : public XrdAccAuthorize, public XrdSciTokensHelper,
                     xrd_rules.emplace_back(AOP_Mkdir, path);
                     xrd_rules.emplace_back(AOP_Rename, path);
                     xrd_rules.emplace_back(AOP_Excl_Insert, path);
+		    xrd_rules.emplace_back(AOP_Stat, path);
```

Comment:
The formatting seems odd here. Can you please check that there was no mixing of tabs/spaces? Thank you.

Reply:
thanks, there was a mixing of tabs when I edited it. amended